.github/workflows/release.yaml

```
@@ -0,0 +1,207 @@
+        uses: actions/checkout@v4
+
+      - uses: bufbuild/buf-setup-action@v1.50.0
+
+      - name: Check proto linting
+        run: buf lint
+
+      - name: Publish protos
+        uses: bufbuild/buf-push-action@v1        
+        with:
+          input: api-spec/protobuf  
+        buf_token: ${{ secrets.BUF_TOKEN }}
```

actionlint (1.7.7)

unexpected key "buf_token" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)